.dir-locals.el

```diff
@@ -1,14 +1,20 @@
 ((nil . ((tab-width . 8)
          (sentence-end-double-space . t)
-         (fill-column . 70)))
+         (fill-column . 70)
+         (projectile-project-configure-cmd . "CFLAGS='-O0 -g3' ./configure --enable-checking='yes,glyphs' --enable-check-lisp-object-type --enable-rust-debug")
+         (projectile-project-compilation-cmd . "make -j16")
+         (projectile-project-test-cmd . "make check")
+         (projectile-project-run-cmd . "./src/remacs")))
```

Comment:
Could you remove your changes to this file? They seem specific to you (my little laptop wouldn't benefit from `-j16`, for example :) ).